javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,16 @@
+var fullName = 'Sara Roohan';
+var isCool = true;
+var totalPets = 0;
+var unknown = null;
+var never;
+
+console.log('value of Full Name:', fullName);
+console.log('value of Is Cool:', isCool);
+console.log('value of Total Pets:', totalPets);
+console.log('typeof fullName:', typeof 'Sara Roohan');
+console.log('typeof isCool:', typeof true);
+console.log('typeof totalPets:', typeof 0);
```

Comment:
you want too place the actual variables here in the console logs.

Reply:
Updated!
![image](https://user-images.githubusercontent.com/78943858/112730071-b41a9880-8eec-11eb-815a-f1be086f4149.png)